crates/core/src/factories.rs

```diff
@@ -8,3 +8,6 @@ pub use core_space::*;
 
 mod mem_peer_store;
 pub use mem_peer_store::*;
+
+mod stub_transport;
```

Comment:
Should this be a `mem_transport`? It does do transport work, just not networking.

Reply:
My thought here was to use "stub" as a distinguisher for module implementations that we don't expect to be production-ready. So when we move your mem-op-store into the core crate, I was going to suggest renaming it to Stub. Totally open to other naming suggestions.

Reply:
Understood. My default would probably be to name them as closely as possible to what they do and solve the "where to use it" question with documentation. People should read what a type is for and what its limitations are before using it ideally.

If we want different names so that there aren't a mixture of "memory" things that should and shouldn't be used in production then my thoughts: Stub to me has no depth to it, like a "shuffle" that just returns its input or a "random_bytes" that just returns a zeroed array. This is a valid implementation that should only be used in a single process. So if not "MemoryXXX" then perhaps "IntraProcessXXX" or something like that? I'm not sure about the op store in this case, I don't really fancy "NoPersistenceOpStore"

Reply:
That's fair. I spent the requisite 2 minutes trying to think of other names and didn't come up with anything better. Let's go with your original suggestion of just using Mem for this and your op store. We can always change them later if needed.

Reply:
Done: https://github.com/holochain/kitsune2/pull/45/commits/f3fbf7b5e92e13992725de3a3f3b6781787ecc92

Reply:
I'd genuinely rather debug a memory leak than come up with names for things :) This looks good, thank you!